<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BallisticUI</class>
 <widget class="QMainWindow" name="BallisticUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BallisticCalculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>251</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="angleBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="initialVelocityBox">
       <property name="toolTip">
        <string extracomment="Initial velocity of projectile in m/s"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="initialHeightBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="projectileMassBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="projectileDiameterBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="cdBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="resultBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="angleLabel">
       <property name="text">
        <string>Angle (degrees)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="velocityLabel">
       <property name="text">
        <string>Initial Velocity (m/s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Initial Height (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="massLabel">
       <property name="text">
        <string>Projectile Mass (kg)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="diameterLabel">
       <property name="text">
        <string>Projectile diameter (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dragLabel">
       <property name="text">
        <string>Base Drag Coefficient</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Range (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="calculateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>82</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Range</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Angle</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
